analysis/language/ja/ja_morph_kagome.go

```diff
@@ -15,23 +15,27 @@ import (
 	"github.com/blevesearch/bleve/analysis"
 	"github.com/blevesearch/bleve/registry"
 
-	"github.com/ikawaha/kagome"
+	"github.com/ikawaha/kagome/tokenizer"
 )
 
 const TokenizerName = "kagome"
 
 type KagomeMorphTokenizer struct {
-	tok *kagome.Tokenizer
+	tok tokenizer.Tokenizer
+}
+
+func init() {
+	_ = tokenizer.SysDic() // prepare system dictionary
```

Comment:
On error this will panic but I think that is okay. It does not make sense to recover if the analysis tool chain initialization failed.